### PR TITLE
docs: create "add a network" post

### PR DESCRIPTION
**Pull request by thlmenezes**
## Description of change

Includes draft posts for upcoming features

1. Creates Network section
1. Creates "Add a Network" post

## Issues resolved by this PR

- [x] [DVN-11821]
- [x] [DVN-11822]

## More info

Post Preview: Networks introduction
image

Post Preview: Add a Network
image

[DVN-11821]: https://devopness.atlassian.net/browse/DVN-11821?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-11822]: https://devopness.atlassian.net/browse/DVN-11822?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/networks/add-network.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    featured:
+required_permissions:
+    - network:create
+---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Networks` card
+1. Click `View` in the `Networks` card to see a list of existing `Networks`
+1. On the upper-right corner of the list click `ADD NETWORK`
+1. Select a connected `Source Provider` account
```
[the remark this answers is not in the document]
Comment (thlmenezes) [reply docs/networks/add-network.md:21]: Same instruction as step 5 from [Add an Application](https://www.devopness.com/docs/applications/add-application/) post in production, validated with users.

Could you be more precise on where the discomfort/confusion with the phrase lies?

File: docs/networks/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+draft: true
+title: Networks
+intro: In this section you will learn how to provision and manage your networks/virtual private cloud (VPC)
```
Comment (jfoliveira) [inline docs/networks/index.md:4]:
```suggestion
intro: Learn how to provision and manage cloud networks, also known as Virtual Private Clouds (VPCs)
```

File: docs/networks/add-network.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+draft: true
+title: Add a Network
+intro: By default, every cloud provider provisions at least one network/VPC (Virtual Private Cloud) in each region to which a user's account has access. However there are cases where you might need to define custom network topology, for example to isolate resources that should not be directly connected. Add a network to define a custom network topology.
```
Comment (jfoliveira) [inline docs/networks/add-network.md:4]: As we support multiple cloud providers, it's too risky to use the `intro` of a topic to try to explain how things work in cloud providers in general.
The assumption made in previous text that "every cloud provider ... at least one VPC ..." is too bold, as some CSPs provision network per region, others do it globally, other CSPs that we still do not support might adopt a different approach.

Please let's just keep things simple, as we've been trying in most topics of our documentation.

Let's go straight to the point, say what a user can do, and perhaps a quick example of when that could be helpful. And that's it. Intro texts should be short.

```suggestion
intro: Add a network to your environment when you need to define custom network topology, so you can have resources (such as Servers and Applications) deployed to different networks, will be completely isolated from each other.
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/networks/add-network.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jfoliveira) [reply docs/networks/add-network.md:21]: @thlmenezes why a user would need a "source provider" to provision a network, which is a "cloud resource"?

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/networks/add-network.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jfoliveira) [reply docs/networks/add-network.md:21]: Network is a cloud resource as much as a server is, so should follow the same workflow as we use to provision a server, choose cloud provider credentials, etc.

[the remark this answers is not in the document]
Comment (thlmenezes) [reply docs/networks/add-network.md]: Sorry, my mistake; fixed in the latest update

**Comment by thlmenezes** [comment]
@jfoliveira while fixing the links for the "Add a Network" post, found some wrong urls to "Add Cloud Provider Credential" post and added to the PR.

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**